lib/proto.js

```diff
@@ -64,7 +64,8 @@ const prototype = {
 
 Object.setPrototypeOf(prototype, EventEmitter.prototype)
 
-module.exports = prototype
+// exporting and consuming the prototype object using factory pattern fixes scoping issues with getters when serializing
+module.exports = () => prototype
```

Comment:
I'm not clear how this is any different. It's still returning the single prototype object. Wouldn't we instead want to do?:

```js
module.exports = function () {
  return Object.create(prototype)
}
```

Reply:
Thank you. Yes, that makes sense. I _think_ this still works for the reported bug because the scope of where the factory function is called - but a new instance from the factory is better.